src/Test/Unit/Config/Validator/Deploy/DeprecatedVariablesTest.php

```diff
@@ -82,45 +94,59 @@ public function executeDataProvider(): array
     {
         return [
             [
+                [],
                 [],
                 [],
                 Success::class,
             ],
             [
                 [DeployInterface::VAR_VERBOSE_COMMANDS => '-v'],
                 [],
+                [],
                 Success::class,
             ],
             [
                 [DeployInterface::VAR_VERBOSE_COMMANDS => Environment::VAL_ENABLED],
                 [],
+                [],
                 Error::class,
             ],
             [
                 [DeployInterface::VAR_STATIC_CONTENT_THREADS => 3],
                 [],
+                [],
                 Error::class,
             ],
             [
+                [],
                 [],
                 [DeployInterface::VAR_STATIC_CONTENT_THREADS => 3],
                 Error::class,
             ],
             [
                 [DeployInterface::VAR_DO_DEPLOY_STATIC_CONTENT => 1],
                 [],
+                [],
                 Error::class,
             ],
             [
+                [],
                 [],
                 [DeployInterface::VAR_DO_DEPLOY_STATIC_CONTENT => 1],
                 Error::class,
             ],
             [
+                [],
                 [DeployInterface::VAR_SCD_EXCLUDE_THEMES => 'theme'],
```

Comment:
add cases with empty value for VAR_SCD_EXCLUDE_THEMES and true value for VAR_STATIC_CONTENT_SYMLINK